checks/Instance.Tests.ps1

```
@@ -175,6 +175,40 @@ $Tags = Get-CheckInformation -Check $Check -Group Instance -AllChecks $AllChecks
         }
     }
 
+    Describe "Service Account Access Level" -Tags ServiceAccountAccess, CIS, Medium, $filename {
+        if ($NotContactable -contains $psitem) {
+            Context "Testing Service Account Access Level on $psitem" {
+                It "Can't Connect to $Psitem" {
+                    $false	|  Should -BeTrue -Because "The instance should be available to be connected to!"
+                }
+            }
+        }
+        else {
+            Context "Testing SQL Service Account Access Level on $psitem" {
+                
+                
+                It "Testing SQL Service Account Access Level on $psitem" {
+
+                    $ServiceAccountEngine =  Get-DbaService -ComputerName $InstanceSMO.ComputerName -InstanceName $InstanceSMO.InstanceName | where ServiceType -eq "Engine" | select StartName
+                    $LocalAdmins = Invoke-Command -ComputerName $InstanceSMO.ComputerName -ScriptBlock { Get-LocalGroupMember -Group "Administrators" } -ErrorAction Stop
```

when adding this part to the AllInstanceInfo function please include try catch so it can silently fail and give a test result of 

"We were unable to contact $ComputerName"